### DISCUSSION TITLE
Entertainment Ideas

### DISCUSSION THREAD

**Discussion by AndreaGriffiths11** [gittogethers/octogatosconf22#7] [General]
Please drop your ideas here

**Comment by pachicodes** [comment]
For the PT track, 
1. Stand-up show with comedian who makes jokes about developers and techs.
https://pokemaobr.dev/
2. A wellness break, something like Chair Yoga, or stretching,
3. Live DJ for closing

**Comment by hafiscp** [comment]
Some Entertainment Ideas,

1. Live Quizzes, either Fun or Interactive ( Kahoot / Quizziz / ... )
2. Meeting with Strangers (Random Discord Server Meets or Something)
3. Competitions which any one can participate and useful in future( Like SparkAR Filter Making, ... )
4. Watch Party or something.
5.
